pkg/quotaplugins/quota-simple-rest/quota_rest_manager.go

```diff
@@ -261,7 +261,9 @@ func (qm *QuotaManager) Fits(aw *arbv1.AppWrapper, awResDemands *clusterstateapi
 	proposedPreemptions []*arbv1.AppWrapper) (bool, []*arbv1.AppWrapper, string) {
 
 	// Handle uninitialized quota manager
+	// If a url does not exists then assume fits quota
 	if len(qm.url) <= 0 {
```

Comment:
nit: I understand it's already there, but while we are at it, changing for equality comparison seems more idiomatic here.